assets/src/components/nav/navMenu.tsx

```diff
@@ -45,59 +43,50 @@ const NavMenu: React.FC<Props> = ({ mobileMenuIsOpen, toggleMobileMenu }) => {
             <OldCloseIcon className="c-nav-menu__close-icon" />
           </button>
         </div>
-        <ul className="c-nav-menu__links">
-          <li>
-            <button
-              className="c-nav-menu__button"
-              onClick={reload}
-              title="Refresh"
-            >
-              <RefreshIcon className="c-nav-menu__icon" />
-              Refresh
-            </button>
-          </li>
-          <li>
-            <button
-              className="c-nav-menu__button"
-              onClick={() => {
-                openDrift()
-                toggleMobileMenu()
-              }}
-              title="Support"
-            >
-              <SpeechBubbleIcon className="c-nav-menu__icon" />
-              Support
-            </button>
-          </li>
-          <li>
-            <button
-              className="c-nav-menu__button"
-              onClick={() => {
-                displayHelp(location)
-                toggleMobileMenu()
-              }}
-              title="About Skate"
-            >
-              <QuestionMarkIcon className="c-nav-menu__icon" />
-              About Skate
-            </button>
-          </li>
-
-          <li>
-            <NavLink
-              className={({ isActive }) =>
-                "c-nav-menu__link" +
-                (isActive ? " c-nav-menu__link--active" : "")
-              }
-              title="Settings"
-              to="/settings"
-              onClick={toggleMobileMenu}
-            >
-              <SettingsIcon className="c-nav-menu__icon" />
-              Settings
-            </NavLink>
-          </li>
-        </ul>
+        <div className="p-3">
+          <Nav className="flex-column" as="ul">
+            <Nav.Item>
+              <Nav.Link as="button" onClick={reload} className="icon-link">
+                <BsIcon.ArrowClockwise /> Refresh
```

Comment:
Related to [Eddie's comment](https://github.com/mbta/skate/pull/2392/files#r1489564125) on the logout-button PR, it might make sense to set _all_ of the icons to aria-hidden.

Reply:
@lemald @joshlarson Okay so I could add `aria-hidden` to each of these icons in the `navMenu` or I could set `aria-hidden` as default on all the icons in `bsIcons` and add that to the instructions of things to do. Since we use `{...props}` on those SVG's, `aria-hidden` can be disabled by specifying `aria-hidden={false}`. Any objections/approvals?

Reply:
I went ahead and did it, let me know if you wanna revert it :) d8de6b7e6f098ff0c54dcbd530a0d3cb561e190f

Reply:
Yeah, I think that's fine since it will be appropriate for most of the places that we end up using these icons.

Reply:
Love it!